packages/javascript-sdk/src/http-client/helpers.ts

```diff
@@ -31,13 +31,13 @@ export function addAuthzInfoToHeaders(
   const headers = new Headers(init.headers);
 
   if (advices.AuthenticateToServiceConditionAdvice) {
-    headers.set('x-tree', advices.AuthenticateToServiceConditionAdvice[0]);
+    headers.set('X-Tree', advices.AuthenticateToServiceConditionAdvice[0]);
   } else if (advices.TransactionConditionAdvice) {
-    headers.set('x-txid', advices.TransactionConditionAdvice[0]);
```

Comment:
I can see in our mock API we refer to "x-txid" e.g. [routes.resource.js](https://github.com/ForgeRock/forgerock-javascript-sdk/blob/fix_sdk-headers_SDKS-2141/e2e/mock-api/src/app/routes.resource.js#L31) - should we also update the header name here?

Reply:
Functionally, the official spec says headers are case insensitive. I can make that change, but I don't believe it's necessary.